src/containers/main-page/cv-form/components/fields/projects/components/CategoryItem.tsx

```diff
@@ -0,0 +1,17 @@
+import ProfiencyItem from 'common/components/profiency/ProfiencyItem';
+import { memo } from 'react';
+
+const CategoryItem = function ({
+  category, skill, tool, onDelete,
+}: any): JSX.Element {
+  const tools = tool?.length ? tool.join(', ') : '';
+  return (
+    <ProfiencyItem
+      headText={`${category} : ${skill} ( ${tools} )`}
```

Comment:
MAybe better to repalce it into the variable?

Reply:
const itemHeadText = (`${category} : ${skill} ( ${tools} )`);

headText={itemHeadText}